### DISCUSSION TITLE
New languages/versions

### DISCUSSION THREAD

**Discussion by Diheng** [tim-hub/obsidian-bible-reference#17] [Ideas]
Hey Tim, thank you so much for this awesome plugin! I am curious if you have any plan to include more languages/versions of the Bible in your backend API system? Will be interested in helping out if there are tasks that could be distributed. Love this tool and want to make it more available to users who want Bible references in other languages as well!

**Comment by tim-hub** [comment]
Thanks @Diheng , yes

that was originally in the plan, for making that, need to build a bible api gateway to consume other api providers.

**Reply by faikulya** [reply]
Hello, friend!
Great job, thank you very much!
Is it possible to add the Bible in Russian from this website to this plugin? This would be very helpful.

https://www.jw.org/ru/%D0%B1%D0%B8%D0%B1%D0%BB%D0%B8%D0%BE%D1%82%D0%B5%D0%BA%D0%B0/%D0%B1%D0%B8%D0%B1%D0%BB%D0%B8%D1%8F/%D1%83%D1%87%D0%B5%D0%B1%D0%BD%D0%B0%D1%8F-%D0%B1%D0%B8%D0%B1%D0%BB%D0%B8%D1%8F/%D0%BA%D0%BD%D0%B8%D0%B3%D0%B8/

**Comment by netfly** [comment]
May be this API could be a solution for that .... https://scripture.api.bible/ 
(by the way .... awesome plugin. I just whould love to have it for german translations as well)

**Reply by tim-hub** [reply]
Thanks, this API looks like a good option, it looks like containing a lot of versions and languages

**Reply by tim-hub** [reply]
I did some researching too.

For the API/bible provider/services.

## data
- [bible db en](https://github.com/scrollmapper/bible_databases/blob/master/README.md)
- [bible db cn cuv](https://github.com/ElijahLabs/bible)

## api
- [bible api](https://bible-api.com/)
- [digital bible api](https://thedigitalbiblelibrary.org/)
- [Bible_Content api](https://bibliaapi.com/docs/Bible_Content)
- [hackathon bible apis](http://www.hackathon.bible/data.html)
- [scripture bible api](https://scriptu[Biblre.api.bible/)
- [ABibliaDigital.com.br](https://www.abibliadigital.com.br) a RESTful API of the Holy Bible developed with Node.js + Express + MongoDB -[github](https://github.com/marciovsena/abibliadigital)

**Comment by tim-hub** [comment]
More versions are coming

https://github.com/tim-hub/obsidian-bible-reference/releases/tag/1.0.2

**Reply by Gloria-hue** [reply]
Hi Tim? Great work! May I request you to please provide for the nwt version? (@jw.org)

**Reply by Gloria-hue** [reply]
hello Tim-Hub?
am glad you are still developing the app/plugin. Some months ago, I requested for the version of a bible I consider my favorite. I did not know how to get an API as you requested for it. SO SORRY ABOUT THE LONG BREAK.  Here it is . (https://www.jw.org/en/online-help/jw-library/) please do your magic with it too? Am happy for the skills you have put to such an honorable use in this age of 'tech'.

**Comment by PolymathCrowsbane** [comment]
Thank you for your hard work on this. Have you considered the [NET Bible API](https://labs.bible.org/api_web_service)?

**Reply by tim-hub** [reply]
will do some research, will add it to todo

**Reply by tim-hub** [reply]
I tried net bible here,

It does not work, due to CORS, it has `strict-origin-when-cross-origin` so it does not allow other origin client to consume it. :(

**Comment by Vladimir-uni** [comment]
Hello, thank you very much for the plugin. Is it possible to add a version in Russian?

**Reply by tim-hub** [reply]
will do some research for it

**Reply by tim-hub** [reply]
add Russian BTI version

![Uploading image.png…]()

**Reply by tim-hub** [reply]
![image](https://github.com/tim-hub/obsidian-bible-reference/assets/2884320/83649666-e36b-413a-9016-e66b04389565)

**Comment by cjulin6651** [comment]
Any chance the CSB will be added at some point?

**Reply by tim-hub** [reply]
From what I can find, I didn't see a free API for CSB translation.

**Comment by DukwonKang** [comment]
Thank you for developing such a great plugin. 
Could you please add Korean revised version Bible as well?

**Reply by tim-hub** [reply]
Add in the latest release, https://github.com/tim-hub/obsidian-bible-reference/releases/tag/25.06.19

**Comment by DukwonKang** [comment]
Thank you very much!!

God bless you

2025년 6월 19일 (목) 오전 11:00, Tim ***@***.***>님이 작성:
